exchanges/bitfinex/bitfinex.go

```diff
@@ -491,6 +494,72 @@ func (b *Bitfinex) GetSiteListConfigData(ctx context.Context, set string) ([]str
 	return resp[0], nil
 }
 
+// GetSiteInfoConfigData returns site configuration data by pub:info:{AssetType} as a map
+// path should be bitfinexInfoPairs or bitfinexInfoPairsFuture???
+// NOTE: See https://docs.bitfinex.com/reference/rest-public-conf
+func (b *Bitfinex) GetSiteInfoConfigData(ctx context.Context, assetType asset.Item) (map[currency.Pair]order.MinMaxLevel, error) {
+	var path string
+	switch assetType {
+	case asset.Spot:
+		path = bitfinexInfoPairs
+	case asset.Futures:
+		path = bitfinexInfoFuturePairs
+	default:
+		return nil, fmt.Errorf("invalid asset type for GetSiteInfoConfigData: %s", assetType)
+	}
+	url := bitfinexAPIVersion2 + path
+	var resp [][][]any
+
+	err := b.SendHTTPRequest(ctx, exchange.RestSpot, url, &resp, status)
+	if err != nil {
+		return nil, err
+	}
+	if len(resp) != 1 {
+		return nil, errors.New("response did not contain only one item")
+	}
+	pairs := map[currency.Pair]order.MinMaxLevel{}
+	data := resp[0]
+	for i := range data {
+		if len(data[i]) != 2 {
+			return nil, errors.New("response contained a tuple without exactly 2 items")
+		}
+		pairSymbol, ok := data[i][0].(string)
+		if !ok {
+			return nil, fmt.Errorf("could not convert first item in SiteInfoConfigData to string: Type is %T", data[i][0])
+		}
+		if strings.Contains(pairSymbol, "TEST") {
+			continue
+		}
+		// SIC: Array type really is any. It contains nils and strings
+		info, ok := data[i][1].([]any)
+		if !ok {
+			return nil, fmt.Errorf("could not convert second item in SiteInfoConfigData to []any; Type is %T", data[i][1])
+		}
+		if len(info) < 5 {
+			return nil, errors.New("response contained order info with less than 5 elements")
+		}
+		minOrder, err := convert.FloatFromString(info[3])
+		if err != nil {
+			return nil, fmt.Errorf("could not convert MinOrderAmount: %s", err)
+		}
+		maxOrder, err := convert.FloatFromString(info[4])
+		if err != nil {
+			return nil, fmt.Errorf("could not convert MaxOrderAmount: %s", err)
+		}
+		pair, err := currency.NewPairFromString(pairSymbol)
+		if err != nil {
+			return nil, err
+		}
+		pairs[pair] = order.MinMaxLevel{
+			Asset:             assetType,
+			Pair:              pair,
+			MinimumBaseAmount: minOrder,
```

Comment:
Could you please add `AmountStepIncrementSize: minOrder`, 

Also I tried to quickly approximate the step a little bit better but I haven't tested it live, you might have some better ideas this is not a change request on this PR though:

```go
func calculateStepSize(infer float64) float64 {
	stepSize := 1.0
	for infer < 1 {
		stepSize /= 10.0
		infer *= 10.0
	}
	return stepSize
}
```

<img width="203" alt="image" src="https://github.com/thrasher-corp/gocryptotrader/assets/9391715/56214f35-042d-494d-bb34-bdb048897234">

Reply:
I have some code to get the quote step size from the orderbook. It's not great though.
I don't think that we should have it in gct.

I can't see we normally infer AmountStepIncrementSize from Base minOrder?
If we don't usually do it, I don't think we should set precedent.
It's definite possible it might NOT be the step size, right?
I don't think it's right for us to. It should be an exercise for the consumer, again.
Can provide functions for it if necessary, named clearly.

Reply:
@gbjk I think you are correct, inferring it at an orderbook level is going to be a better approach.

Reply:
Based on a discussion, leaving this for a separate PR